app/views/cookies/cookie_details.html.haml

```diff
@@ -0,0 +1,119 @@
+= govuk_page_title(t('cookies.page_title', service_name: service_name))
+
+%p.govuk-body= t('cookies.section_one.paragraph_one')
+
+%p.govuk-body= t('cookies.section_one.paragraph_two')
+
+%h2.govuk-heading-l= t('cookies.section_necessary.section_heading')
+
+%p.govuk-body= t('cookies.section_necessary.paragraph_one')
+
+%table.govuk-table
+  %caption.govuk-table__caption.govuk-visually-hidden
+    = t('cookies.section_necessary.table.caption')
+  %thead.govuk-table__head
+    %tr.govuk-table__row
+      %th.govuk-table__header{scope: 'col'}
+        = t('cookies.section_necessary.table.heading_one')
+      %th.govuk-table__header{scope: 'col'}
+        = t('cookies.section_necessary.table.heading_two')
+      %th.govuk-table__header{scope: 'col'}
+        = t('cookies.section_necessary.table.heading_three')
+  %tbody.govuk-table__body
+    %tr.govuk-table__row
+      %td.govuk-table__cell
+        = t('cookies.section_necessary.table.row_one_a')
+      %td.govuk-table__cell
+        = t('cookies.section_necessary.table.row_one_b')
+      %td.govuk-table__cell
+        = t('cookies.section_necessary.table.row_one_c')
+
+%h2.govuk-heading-l= t('cookies.section_functional.section_heading')
+
+%p.govuk-body= t('cookies.section_functional.paragraph_one')
+
+%table.govuk-table
+  %caption.govuk-table__caption.govuk-visually-hidden
+    = t('cookies.section_functional.table.caption')
+  %thead.govuk-table__head
+    %tr.govuk-table__row
+      %th.govuk-table__header{scope: 'col'}
+        = t('cookies.section_functional.table.heading_one')
+      %th.govuk-table__header{scope: 'col'}
+        = t('cookies.section_functional.table.heading_two')
+      %th.govuk-table__header{scope: 'col'}
+        = t('cookies.section_functional.table.heading_three')
+  %tbody.govuk-table__body
+    %tr.govuk-table__row
+      %td.govuk-table__cell
+        = t('cookies.section_functional.table.row_one_a')
+      %td.govuk-table__cell
+        = t('cookies.section_functional.table.row_one_b')
+      %td.govuk-table__cell.app-no-wrap
```

Comment:
Could `.app-no-wrap` styles be applied to all table cells and forego the custom class?